compiler/modules/front.syntax/scan.bal

```diff
@@ -61,6 +61,30 @@ function scanLineFragIndex(ScannedLine line, int codePointIndex) returns [int, i
     return [fragCodeIndex, fragmentIndex];
 }
 
+function scanLineToString(ScannedLine line) returns string {
```

Comment:
Let's move this function and the previous one down in the file to just before unicodeEscapeValue.

Reply:
fixed